third_party/docfx/templates/devsite/partials/uref/namespace.tmpl.partial

```diff
@@ -1,7 +1,9 @@
 {{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
 
 {{#alt_link}}
-<aside class="note"><strong>Note:</strong> You can also view this documentation at <a href="{{alt_link}}" class="external">{{alt_link}}</a>.</aside>
+<aside class="note">
+  <strong>Note:</strong>To get more information about this package, such as access to older versions, view <a href="{{alt_link}}" class="external">this package on pkg.go.dev</a>.

```

Comment:
will `alt_link` only be used for Go or would we possibly have to come back and edit this to make it not as language specific?

Reply:
Good call out! We added it; it's only used for Go. So, if we use it in another language, we'd need to update this.

I looked, but unfortunately there doesn't seem to be an easy way to branch the template based on language. I thought there was, but I'm not seeing it at the moment.